<launch>
  <!-- camera driver -->
  <node name="davis_ros_driver" pkg="davis_ros_driver" type="davis_ros_driver" />
  
  <!-- complementary filter -->
  <node name="reconstruction_node" pkg="complementary_filter" type="reconstruction_node" output="screen" args="--v=3">
    <param name="global_log_intensity_state_update_frequency" type="double" value="30.0"/>
    <param name="contrast_threshold_recalibration_frequency" type="double" value="10.0"/>

    <remap from="events" to="dvs/events" />
    <remap from="image_raw" to="dvs/image_raw" />
  </node>

  <!-- display -->
  <node name="image_view" pkg="image_view" type="image_view">
    <remap from="image" to="complementary_filter/intensity_estimate"/>
  </node>

  <!-- configure -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

</launch>
